app/bundles/UserBundle/Resources/views/Helper/publishstatus_icon.html.twig

```diff
@@ -0,0 +1,98 @@
+{#
+  Variables
+    * item (object)
+    * model (string)
+    * query (optional, string)
+    * size (optional, string)
+    * onclick (optional, string)
+    * attributes (optional, array)
+    * transKeys (optional, array)
+#}
+{# set defaults #}
+{%- set query = query|default('') %}
+{%- set size = size|default('fa-lg') %}
+{%- set onclick = onclick|default('') %}
+{%- set attributes = attributes|default({}) %}
+{%- set transKeys = transKeys|default({}) %}
+{%- set status = status|default(item.publishStatus) %}
+{%- set text = text|default('') %}
+{%- set icon = text|default('') %}
+
+{%- if query is not empty %}
+  {%- set queryParam = query|parse_str %}
+  {%- if queryParam.customToggle is defined %}
+    {# needs more testing #}
```

Comment:
What does it mean?

Reply:
Please do not resolve comments that aren't resolved :)

This file is still just a copy of the one in the CoreBundle as @kuzmany mentioned in another comment. Let's not copy paste code around. I can see this is the only change:
![Screenshot 2024-02-27 at 12 52 42](https://github.com/mautic/mautic/assets/1235442/88473ed2-27f3-43e2-bbcb-f76f0dd7e3ab)
I'd rather use the publish/unpublish labels than duplicate the whole template.